src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -14,6 +14,8 @@
 using System.Runtime.InteropServices;
 using UIAutomationClient;
 
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("AutomationTests")]
```

Comment:
This will fail in the signed build pipeline. Please see https://github.com/microsoft/axe-windows/blob/main/docs/AccessingInternals.md for info on how to make internals visible

Reply:
Thanks, updating.

Reply:
Added in latest iteration